static/app/views/settings/organizationAuthTokens/index.spec.tsx

```diff
@@ -0,0 +1,290 @@
+import {initializeOrg} from 'sentry-test/initializeOrg';
+import {
+  render,
+  renderGlobalModal,
+  screen,
+  userEvent,
+  waitFor,
+} from 'sentry-test/reactTestingLibrary';
+import {textWithMarkupMatcher} from 'sentry-test/utils';
+
+import * as indicators from 'sentry/actionCreators/indicator';
+import OrganizationsStore from 'sentry/stores/organizationsStore';
+import {OrgAuthToken} from 'sentry/types';
+import {OrganizationAuthTokensIndex} from 'sentry/views/settings/organizationAuthTokens';
+
+describe('OrganizationAuthTokensIndex', function () {
+  const ENDPOINT = '/organizations/org-slug/org-auth-tokens/';
+  const PROJECTS_ENDPOINT = '/organizations/org-slug/projects/';
+  const {organization, project, router} = initializeOrg();
+
+  const defaultProps = {
+    organization,
+    router,
+    location: router.location,
+    params: {orgId: organization.slug},
+    routes: router.routes,
+    route: {},
+    routeParams: router.params,
+  };
+
+  let projectsMock: jest.Mock<any>;
+
+  beforeEach(function () {
+    OrganizationsStore.addOrReplace(organization);
+
+    projectsMock = MockApiClient.addMockResponse({
+      url: PROJECTS_ENDPOINT,
+      method: 'GET',
+      body: [project],
+    });
+  });
+
+  afterEach(function () {
+    MockApiClient.clearMockResponses();
+  });
+
+  it('shows tokens', async function () {
+    const tokens: OrgAuthToken[] = [
+      {
+        id: '1',
+        name: 'My Token 1',
+        tokenLastCharacters: '1234',
+        dateCreated: new Date('2023-01-01T00:00:00.000Z'),
+        scopes: ['org:read'],
+      },
+      {
+        id: '2',
+        name: 'My Token 2',
+        tokenLastCharacters: 'ABCD',
+        dateCreated: new Date('2023-01-01T00:00:00.000Z'),
+        scopes: ['org:read'],
+        dateLastUsed: new Date(),
+        projectLastUsedId: project.id,
+      },
+    ];
+
+    const mock = MockApiClient.addMockResponse({
+      url: ENDPOINT,
+      method: 'GET',
+      body: tokens,
+    });
+
+    render(<OrganizationAuthTokensIndex {...defaultProps} />);
+
+    // First shows loading indicator
+    expect(await screen.findByTestId('loading-indicator')).toBeInTheDocument();
```

Comment:
we could also use here

`await waitForElementToBeRemoved(() => queryByTestId('loading-indicator'))`

Reply:
nice!